<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MyField" name="widget_1" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>450</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>450</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MyField" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>450</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>450</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="addEdgeComboBox1_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="addEdgeComboBox1_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="addEdgeComboBox2_1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>540</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="addEdgeComboBox2_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>540</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addEdgeButton_1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>540</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Edge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addEdgeButton_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>540</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Edge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addVertexButton1_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Vertices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addVertexButton1_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>490</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop add vertices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addVertexButton2_1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add vertices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addVertexButton2_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop add vertices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>570</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compIzoButton1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>610</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Oblicz izomorfizm</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textIso">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>610</y>
      <width>256</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="giComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton_1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>490</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyField</class>
   <extends>QWidget</extends>
   <header>myfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
